app/renderer/js/utils/enterprise-util.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
```

Comment:
This change here should have been done in the first commit; I ended up reviewing a change that you end up removing in the end.